### PR TITLE
fix: trim whitespace from a string multiple choice question

### PR DESCRIPTION
**Pull request by scissorsneedfoodtoo**
Checklist:

- [ ] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [ ] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [ ] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

This PR adjusts the spacing for one of the distractors and the correct answer for q3.

### CODE REVIEW

**Review by Sembauke — APPROVED** [review]

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
image
You will need to use codeblocks, with the in-line code multiple spaces collapse to one

File: curriculum/challenges/english/25-front-end-development/lecture-working-with-common-string-methods/67326c3c3ab931c644cea05b.md [hunk 1]
```diff
@@ -103,11 +103,11 @@ Consider which part of the string is affected by `trimEnd()`.
 
 ---
 
-`" Code"`
+`"   Code"`
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lecture-working-with-common-string-methods/67326c3c3ab931c644cea05b.md:106]:
````suggestion
```js
"   Code"
```
````

File: curriculum/challenges/english/25-front-end-development/lecture-working-with-common-string-methods/67326c3c3ab931c644cea05b.md [hunk 2]
```diff
@@ -103,11 +103,11 @@ Consider which part of the string is affected by `trimEnd()`.
...
 
 ---
 
-`"Code "`
+`"Code   "`
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lecture-working-with-common-string-methods/67326c3c3ab931c644cea05b.md:110]:
````suggestion
```js
"Code   "
```
````

**Review by majestic-owl448 — APPROVED** [review]

**Comment by scissorsneedfoodtoo** [comment]
Sorry, just seeing this now. Thanks for catching that and for the fix, @ilenia-magoni!

**Pull request merged**